app/models/commit.rb

```diff
@@ -65,7 +65,7 @@ def self.get_committer_from_github(repo, r_commit)
     github = Rails.configuration.x.github
     commit = github.repos.commits.get(Rails.application.config.organisation,
                                       repo.name, r_commit.oid)
-    if commit.committer
+    if commit.committer and commit.author
```

Comment:
I think you can change this to `if commit.author` instead, because the [the following commit](https://github.com/ZeusWPI/gamification/commit/54c9b47843774ef284b192d912bb8218d443701b) did change from commiter to author.